sdl-core/src/main/scala/io/smartdatalake/util/misc/DataFrameUtil.scala

```diff
@@ -239,8 +238,9 @@ private[smartdatalake] object DataFrameUtil {
     def isEqual(df2: DataFrame): Boolean = {
       // As a set-theoretic function symmetricDifference ignores multiple occurences of the same row.
       // Thus we need also to compare the cardinalities and the schemata of the two data frames.
+      // For the schema the order of columns doesnt need to much.
```

Comment:
```suggestion
      // For the schema, the order of columns doesn't need to match.
```

Reply:
wow, thats too match in one sentence ;-)